app/src/main/res/layout/fragment_previous_insurer.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
We've been moving towards using `previous_insurer_fragment` over `fragment_previous_insurer`, given the codegen from `ViewBinding` generating `PreviousInsurerFragmentBinding` instead of `FragmentPreviousInsurerFragment` 🙂

Reply:
Ah, very reasonable!